drivers/include/periph/wdg.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2016 Unwired Devices [info@unwds.com]
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup
+ * @ingroup
+ * @brief
+ * @{
+ * @file		wdg.h
+ * @brief       Common watchdog peripheral interface
+ * @author      EP [ep@unwds.com]
+ */
+#ifndef DRIVERS_INCLUDE_PERIPH_WDG_H_
+#define DRIVERS_INCLUDE_PERIPH_WDG_H_
+
+#include <stdbool.h>
+
+/**
+ * @brief Sets watchdog timer prescaler.
+ *
+ * @param prescaler watchdog timer prescaler value
```

Comment:
Please add `[in]` or `[out]` for all the function parameters to specify if they are input or output parameters.